### DISCUSSION TITLE
Extension Methods on Static Interfaces

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/csharplang#6739] [Language Ideas] [answered]
I'd like to request the ability to create extension methods for static intefaces.  For example:

```
    public static class StaticExtensions {
        public static bool ParseCustom<T>(static this T, string Input) where T : IParsable<T> {
            //
        }
    }
```

**Comment by svick** [comment, marked as answer]
Even though the motivation is a bit different, this is a duplicate of https://github.com/dotnet/csharplang/discussions/2505 and more generally part of https://github.com/dotnet/csharplang/issues/192.

**Reply by TahirAhmadov** [reply]
Isn't it superseded by #5497 ?
